recipes/sobjectizer/all/conanfile.py

```diff
@@ -28,79 +32,86 @@ class SobjectizerConan(ConanFile):
     }
 
     exports_sources = ["CMakeLists.txt"]
```

Comment:
```suggestion
```
With `CMakeToolchain` generator, the current `CMakeLists.txt` is no longer requires and should be removed from this package. Please, remove that file sobjectizer folder.

Reply:
@uilianries this PR is to only add new version (new library release), not to clean up unused files. it can be done safely in another PR. in general, it's good practice to keep PR focused on one change only and avoid doing several unrelated changes within same PR.

Reply:
This PR also largely deals with migration to Conan v2 (despite the PR name), so removing a file left over from the migration is still relevant here, I would say.

Reply:
okay, please update title to reflect all the actual changes made within PR